lib/rules/lines-around-comment.js

```diff
@@ -207,12 +202,7 @@ module.exports = {
          * @returns {boolean} True if the comment is at parent end.
          */
         function isCommentAtParentEnd(node, nodeType) {
-            const ancestors = context.getAncestors();
-            let parent;
-
-            if (ancestors.length) {
-                parent = ancestors.pop();
-            }
+            const parent = sourceCode.getNodeByRangeIndex(node.range[0]);
```

Comment:
Yeah, that's a good point. And looks to be the only place we're using it, so a comment would work!